### PR TITLE
Add Clinker package

### PR DESCRIPTION
**Pull request by lonsbio**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/clinker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - samtools
+    - bpipe
+    - star 2.5.3a.*
+    - python 2.7.*
```
Comment (bgruening) [inline recipes/clinker/meta.yaml]: python should not be pinned like this. Please use the build/skip section for this like `skip: True  # [py3k]`

File: recipes/clinker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  md5: "5487f99af6b9499c87d4d945e9cd9790"
+
+build:
+  number: 0
+
+requirements:
+  run:
+    - samtools
+    - bpipe
+    - star 2.5.3a.*
```
Comment (bgruening) [inline recipes/clinker/meta.yaml]: Is this strict star pinning necessary?

File: recipes/clinker/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+
+
+test:
+  commands:
+    - bpipe --help
+
+about:
+  home: https://github.com/Oshlack/Clinker
+  summary: "test"
```
Comment (bgruening) [inline recipes/clinker/meta.yaml]: Can you please add here a slightly more verbode summary :)

File: recipes/clinker/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  license: GPL3
```
Comment (bgruening) [inline recipes/clinker/meta.yaml]: license_family: GPL

Comment (bgruening) [reply recipes/clinker/meta.yaml]: license_file: LICENSE

File: recipes/clinker/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    - bpipe
+    - star 2.5.3a.*
+    - python 2.7.*
+
+
+
+
+test:
+  commands:
+    - bpipe --help
```
Comment (bgruening) [inline recipes/clinker/meta.yaml]: This does not look like a clinker test :)

Comment (lonsbio) [reply recipes/clinker/meta.yaml]: Clinker is invoked via bpipe - but we may need to write a wrapper.

**Comment by lonsbio** [comment]
Thanks @bgruening and apologies  - we're being introduced to creating BioConda packages with @Slugger70  and I got a little ahead of things! Will fix things up.

**Pull request closed without merging**